Havtorn/Source/Engine/Graphics/RenderManager.cpp

```diff
@@ -1205,45 +1412,45 @@ namespace Havtorn
 		Context->PSSetConstantBuffers(0, 1, &FrameBuffer);
 	}
 
-	void CRenderManager::GBufferData(const SRenderCommand& command)
+	void CRenderManager::GBufferData(const SRenderCommand& /*command*/)
```

Comment:
Please add a comment explaining why this GBufferData(...) has its contents commented.